contents.json

```diff
@@ -17502,6 +17502,25 @@
       ],
       "date_added": "Mar 17 2019",
       "suggested_by": "@madyanov"
-    }
+    },
+    {
+    	"title": "Science Journal",
+    	"category-ids": [
+    		"misc"
+    	],
+    	"description": "Gather data around you",
```

Comment:
I think: "Use the sensors in your mobile devices to perform science experiments" would be a more descriptive `description`